### ISSUE TITLE
[Feature Request] Option to subscribe to Playlists

### ISSUE THREAD

**Issue by Grammost** [TeamNewPipe/NewPipe#1376]
It would be great if there was an option to subscribe to playlists, because often times you will find yourself only interested to a specific kind of videos someone uploads.
This would be even more useful, after #1044 is finished.

It could work similar to how you subscribe to channels, for example:
https://www.youtube.com/feeds/videos.xml?playlist_id=PLiuUQ9asub3Ta8mqP5LNiOhOygRzue8kN
If you put this in a RSS Reader, you will only see Hubble videos and not every video NASA uploads.

**Comment by Stypox** [comment]
Closing in favour of #3525
